ci-operator/config/openshift/kueue-operator/openshift-kueue-operator-main.yaml

```diff
@@ -66,6 +59,140 @@ tests:
   skip_if_only_changed: ^\.tekton/|\.md$|^(LICENSE|OWNERS)$
   steps:
     workflow: openshift-ci-security
+- as: test-e2e-4-17
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.17"
+  steps:
+    test:
+    - as: e2e-kueue
+      cli: latest
+      commands: |-
+        echo "Current PWD: $(pwd)"
+        echo "Listing directory contents:"
+        ls -lah
+        REVISION=$(git log --oneline -1 | awk '{print $4}' | tr -d "'")
+        echo "Current Git branch:"
+        git branch --show-current
+
+        echo "Latest Git commits:"
+        git log --oneline -5
+
+        echo "Git status:"
+        git status
+        export OPERATOR_IMAGE=quay.io/redhat-user-workloads/kueue-operator-tenant/kueue-operator-1-0:on-pr-${REVISION}
```

Comment:
This would need to change.

This would probably need to be the konflux main images that you create.

Reply:
There are a few items like this that will change once you have the konflux images.